### PR TITLE
Update Getting Started pages

### LINKED ISSUE #13639: Fix _weakref import error when using LLDB on Darwin
## Reason
The python dylib lldb is currently linking with, is incorrect. This latest update to the package solves this issue: https://github.com/idaholab/package_builder/pull/207

## Design
Update the package_version file so as to update the mooseframework.org links.

## Impact
Updates links on our Getting Started pages

### PR DESCRIPTION
**Pull request by milljm**
A new version of the package is available (fixes _weakref import).
Modify instructions where they were lacking, or otherwise wrong.

Closes #13639

### CODE REVIEW

**Comment by milljm** [comment]
As discovered by @loganharbour, High Sierra users must install Xcode, not just Xcode command line tools as we specify in our Getting Started instructions. Do no merge, until I have updated those instructions.

File: modules/doc/content/getting_started/installation/osx_other.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Mac OS High Sierra or Older
+
+!include getting_started/minimum_requirements.md
+
+## Prerequisites
+
+The following, is required to be installed before you can begin using MOOSE.
+
+Due to the unfortunate difficulty Apple has placed on its users wishing to develop on platforms older than Mojave, it is strongly encouraged users upgrade to Mojave.
```
Comment (loganharbour) [inline modules/doc/content/getting_started/installation/osx_other.md]: Ha.

Comment (dschwen) [reply modules/doc/content/getting_started/installation/osx_other.md]: Downloading the mac OS Catalina Beta to stay ahead of the curve!

Comment (milljm) [reply modules/doc/content/getting_started/installation/osx_other.md]: Wonder if they will ever remove the need to install the headers package

Comment (aeslaughter) [inline modules/doc/content/getting_started/installation/osx_other.md]: I think the following should be at the top of the page.

!alert info title=Consider updating to Mojave
Due to the unfortunate difficulty Apple has placed on users wishing to develop on platforms older than Mojave, it is strongly encouraged users upgrade to Mojave.

**Review by aeslaughter — CHANGES_REQUESTED** [review]

**Review by aeslaughter — DISMISSED** [review]

**Comment by cticenhour** [comment]
I noticed these OSX instructions are now saying that users are required to install Xcode after CLT, but I don't see how we actually need both.....I get the caveat that we need to source CLT from Apple manually if we *already have* Xcode, but that's the only instance I see where we'd explicitly have both on a new user system. Why are we now requiring both here?

**Comment by loganharbour** [comment]
It's needed for lldb to work on systems older than Mojave.

**Comment by cticenhour** [comment]
Sorry, I should have clarified - I was referring to the Mojave instructions.

**Comment by cticenhour** [comment]
Nevermind.....Looks like I didn't go far enough in testing lldb on a clean install of Mojave. Xcode is definitely required 😄

**Comment by milljm** [comment]
A user found an issue with moose_profile I want to fix.

**Comment by permcody** [comment]
Roger - we have other breakage I'm attempting to fix.

**Review by permcody — APPROVED** [review]

**Pull request merged**